features/netsocket/emac-drivers/TARGET_NXP_EMAC/TARGET_IMX/imx_emac.cpp

```diff
@@ -278,8 +282,14 @@ emac_mem_buf_t *Kinetis_EMAC::low_level_input(int idx)
         p = rx_buff[idx];
         memory_manager->set_len(p, length);
 
+#if defined(FSL_SDK_ENABLE_DRIVER_CACHE_CONTROL) && FSL_SDK_ENABLE_DRIVER_CACHE_CONTROL
+        /* Add the cache invalidate maintain. */
+        DCACHE_InvalidateByRange((uint32_t)bdPtr->buffer, g_handle.rxBuffSizeAlign[0]);
```

Comment:
If you are going to be cache invalidating, I believe the RX buffer alignment will need to be increased from 4 to 32 (`FSL_FEATURE_L1DCACHE_LINESIZE_BYTE`)